chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -225,11 +225,14 @@ def _write_records(self, records: Sequence[EmbeddingRecord]) -> None:
                 exists_in_index = self._id_to_label.get(
                     id, None
                 ) is not None or self._brute_force_index.has_id(id)
+                exists_in_bf_index = self._brute_force_index.has_id(id)
 
                 if op == Operation.DELETE:
                     if exists_in_index:
                         self._curr_batch.apply(record)
-                        self._brute_force_index.delete([record])
+                        self._brute_force_index.delete(
```

Comment:
can we not needlessly make this a ternary

Reply:
Absolutely I just didn't want to add another nesting but yeah, None means nothing in this context